### ISSUE TITLE
ios write blob to file

### ISSUE THREAD

**Issue by wowzaaa** [jDataView/jBinary#55]
Hi

I need to write the contents of an image returned from an xhr call with arraybuffer
on android this works but on ios it is a nightmare ... mainly to write the blob to file
can this library help me solve this?

thanks

**Comment by RReverser** [comment]
To which file? JavaScript doesn't allow writing to the filesystem (unless you are in Cordova or some similar environment - that is, not a browser).

**Comment by wowzaaa** [comment]
yeah i am currently doing this in cordova and soo i will use react-native ... but write now i am having problem writing blobs to file on ios ... i thought this library can help


---- On Wed, 14 Sep 2016 00:56:25 +0300 Ingvar Stepanyan &lt;notifications@github.com&gt;wrote ---- 




To which file? JavaScript doesn't allow writing to the filesystem (unless you are in Cordova or some similar environment - that is, not a browser).

**Comment by RReverser** [comment]
Unfortunately, can't help with that - you need to check Cordova's docs https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-file/#ios-quirks, but this library doesn't have any specific workarounds for those environments at the moment.
